wom/src/main/scala/wom/graph/WomIdentifier.scala

```diff
@@ -0,0 +1,36 @@
+package wom.graph
+
+/**
+  * Identifies a node in its local context.
+  * Used in during graph construction to link nodes together and validate the graph.
+  */
+case class LocalName(private val value: String) {
+  def asString: String = value
+  def combineToLocalName(other: String) = LocalName(s"$value.$other")
```

Comment:
Yeah I thought of leaving this abstract and have each language define their `combine` but I convinced myself that the `.` separation could be the `WOM` separation too. Since Cromwell will use those FQNs for metadata etc it would be nice to have a consistent "look" between workflows for FQNs, even from different language.